terraform/aks/variables.tf

```diff
@@ -98,6 +98,7 @@ variable "postgres_flexible_server_sku" { default = "B_Standard_B1ms" }
 variable "postgres_enable_high_availability" { default = false }
 variable "azure_enable_backup_storage" { default = true }
 variable "enable_container_monitoring" { default = false }
+variable "enable_logit" { default = true }
```

Comment:
We'd like to enable it per environment. Can we keep the default false set it to true in the tfvars?
Maybe only the test environments, review, qa, staging

Reply:
done